provisioning/miniprov/hare_mp/main.py

```diff
@@ -201,6 +202,97 @@ def test(args):
         exit(-1)
 
 
+def reset(args):
+    try:
+        rc = 0
+        exit(rc)
+    except Exception as error:
+        logging.error('Error during reset (%s)', error)
+        exit(-1)
+
+
+def cleanup(args):
+    try:
+        rc = 0
+        url = args.config[0]
+        validator = Validator(ConfStoreProvider(url))
+        if validator.is_first_node_in_cluster():
+            if is_cleanup_done():
+                logging.info('No data to cleanup')
+                exit(0)
+
+            nodes = get_all_nodes()
+            rc = config_cleanup(url, nodes) & logs_cleanup(url, nodes)
+
+        exit(rc)
+    except Exception as error:
+        logging.error('Error during cleanup (%s)', error)
+        exit(-1)
+
+
+def is_cleanup_done() -> bool:
+    return not os.path.isfile('/var/lib/hare/consul-agents.json')
+
+
+def logs_cleanup(url: str, nodes: List[str]):
+    try:
+        for node in nodes:
+            if is_localhost(node, url):
+                os.system('rm -rf /var/log/hare')
+            else:
+                os.system('ssh ' + node + ' rm -rf /var/log/hare')
+
+        return 0
+    except Exception as error:
+        logging.error('Error during logs cleanup (%s)', error)
+        return -1
+
+
+def config_cleanup(url: str, nodes: List[str]):
+    try:
+        for node in nodes:
+            if is_localhost(node, url):
+                os.system('rm -rf /var/lib/hare/*')
+            else:
+                os.system('ssh ' + node + ' rm -rf /var/lib/hare/*')
```

Comment:
`/var/lib/hare/*` is a glob. Are you sure that it will be expanded at _remote_ machine? I'm not that sure. Usually bash linters recommend to wrap such expressions into quotes to prevent globbing BEFORE the command is transmitted to the proper machine.